modeling-cmds/src/shared.rs

```diff
@@ -761,6 +761,34 @@ pub enum ExtrudeMethod {
     Merge,
 }
 
+/// Extrusion method determining if the extrusion will be part of the existing object or an
+/// entirely new object.
+#[derive(Clone, Copy, Debug, PartialEq, Serialize, Deserialize, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+#[cfg_attr(feature = "ts-rs", derive(ts_rs::TS))]
+#[cfg_attr(feature = "ts-rs", ts(export_to = "ModelingCmd.ts"))]
+pub enum ExtrudeReference {
+    /// Extrudes to the entity that this UUID represents
+    EntityReference {
+        /// The UUID of the entity to extrude to.
```

Comment:
Shouldn't this be a face? Otherwise, is the extrude going to meet at the bottom of the object, or the top?